honeybee_energy/properties/face.py

```diff
@@ -67,6 +72,28 @@ def construction(self, value):
             value.lock()  # lock editing in case construction has multiple references
         self._construction = value
 
+    @property
+    def vent_crack(self):
+        """Get or set a AFNCrack object to specify the operable portion."""
+        return self._vent_crack
+
+    @vent_crack.setter
+    def vent_crack(self, value):
+        if value is not None:
+            assert isinstance(value, AFNCrack), 'Expected AFNCrack ' \
+                'for Face vent_crack. Got {}'.format(type(value))
+            if value._parent is None:
+                value._parent = self.host
+            elif value._parent.identifier != self.host.identifier:
+                raise ValueError(
+                    '{0} objects can be assigned to only one parent.\n{0} cannot be '
+                    'assigned to Face "{1}" since it is already assigned to "{2}".\n'
+                    'Try duplicating the object and then assign it.'.format(
+                        'AFNCrack', self.host.identifier,
+                        value._parent.identifier))
```

Comment:
@chriswmackey 

I thought it would be useful to treat the cracks the same as the vent openings, but on second thought I think this makes more sense. The AFNCrack won't change for different host geometries or identifiers, but it does change based on surface type, boundary condition, and between rooms, but I think locking and using the same crack will be fine for each of these groups.